apps/src/lab2/projects/ProjectManager.ts

```diff
@@ -320,6 +323,34 @@ export default class ProjectManager {
     this.lastSource = JSON.stringify(lastSource);
   }
 
+  getShouldCaptureThumbnail() {
+    const {isShareView} = getStore().getState().lab;
+    return this.channelId && this.lastChannel?.isOwner && !isShareView;
```

Comment:
could we pass in `isShareView` as part of the project manager constructor rather than fetching it from redux?

Reply:
Additionally, there's no point in capturing a thumbnail if the project is not shareable, right? Should we add that logic to avoid unnecessary saves?

Reply:
I believe we will only capture thumbnail images for standalone lab2 projects which are always shareable.